hack/build-operator-podman.sh

```diff
@@ -8,14 +8,13 @@ fi
 IFS=',' read -r -a PLATFORM_LIST <<< "$PLATFORMS"
 
 # Remove any existing manifest and image
-podman manifest rm "${OPERATOR_IMAGE_TAGGED}" || true
-podman rmi "${OPERATOR_IMAGE_TAGGED}" || true
+podman manifest rm "${OPERATOR_IMAGE_TAGGED}" 2>/dev/null
+podman rmi "${OPERATOR_IMAGE_TAGGED}" 2>/dev/null
```

Comment:
you do need `|| true` please, isnt it ?

Reply:
in anyways it will suppress the error but let me add || true.

Reply:
if it suppress the error in case image / manifest not found,  no need to add, my mistake then
thanks